src/main/kotlin/guru/stefma/androidartifacts/ArtifactsExtension.kt

```diff
@@ -1,6 +1,7 @@
 package guru.stefma.androidartifacts
 
 import org.gradle.api.Action
+import org.gradle.api.publish.maven.MavenPom
```

Comment:
MavenPom is only available at Gradle 4.8+.
I don't have to make this public because I want to support older Gradle versions too...

Reply:
`MavenPom` is available since at least gradle [`v3.0.0`](https://github.com/gradle/gradle/blob/v3.0.0/subprojects/maven/src/main/java/org/gradle/api/publish/maven/MavenPom.java) but with fewer properties. It's not a problem to have it as part of the public API. Starting with [`v4.8.0`](https://github.com/gradle/gradle/blob/v4.8.0/subprojects/maven/src/main/java/org/gradle/api/publish/maven/MavenPom.java) the new properties like `url`, `description` and `licenses` where added.